### PR TITLE
#5 - Create models in the project (Backend) 

### PR DESCRIPTION
**Pull request by ImmortalIdiot**
The first version of models

### CODE REVIEW

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/BaseEntity.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.example.BackendMIIT.models.domain;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+
+@MappedSuperclass
```
Comment (NN171) [inline backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/BaseEntity.java:9]: Set annotation Getter

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Department.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.ToString;
+
+@Entity
+@Table(name = "departments")
+@Data
+@ToString
```
Comment (NN171) [inline backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Department.java]: Data include toString. Remove it

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/BaseEntity.java (hunk 1 above)
Comment (ImmortalIdiot) [reply backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/BaseEntity.java:9]: Done

File: backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Department.java (hunk 2 above)
Comment (ImmortalIdiot) [reply backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Department.java]: Done

**Review by NN171 — APPROVED** [review]
Fine

**Pull request merged**